docs/getting-started/flashing-firmware/nrf52/drag-n-drop.mdx

```diff
@@ -16,7 +16,7 @@ Download and unzip the latest firmware from [Meshtastic Downloads](https://mesht
 1. Connect your device to your computer with a USB data cable.
 2. Double click the reset button on your device (this will put it into bootloader mode)
```

Comment:
For RP2040 you should actually hold the BOOTSEL button while plugging in the cable. Double click doesn't seem to work.